### ISSUE TITLE
I'm surprised that there is no rich text editor

### ISSUE THREAD

**Issue by rsmithlal** [solidusio-contrib/solidus_static_content#68]
Are we expected to provide our own for the static page content? Any recommendation?

**Comment by elia** [comment]
I think in the past that's what happened most of the time. With ActionText available in Rails 6+ would probably make sense to integrate that when available.

Would you be interested into contributing that?

**Comment by memotoro** [comment]
Hello @rsmithlal I'm using `https://github.com/solidusio-contrib/solidus_editor` in  combination with this project to offer a better experience in the admin when editing this static content. That gives you I think two options as editor CKEditor and TinyMCE, so you choose what you prefer and add extra config files for each one of those. I hope this works for you. Regards

**Comment by elia** [comment]
@memotoro good call! I added an issue (https://github.com/solidusio-contrib/solidus_editor/issues/48) to add action-text to solidus_editor 👍

**Comment by rsmithlal** [comment]
Good idea to add support for action text. If I'm not mistaken, the default rich text editor for Rails 6 is Trix. Perhaps you might want to integrate this with Solidus editor in addition to CKEditor and TinyMCE? https://github.com/basecamp/trix
